src/tools/cephfs/cephfs-shell

```diff
@@ -1245,6 +1249,50 @@ sub-directories, files')
                 perror('max_files is not set')
                 pass
 
+    snap_parser = argparse.ArgumentParser(description='Snapshot Management')
+    snap_parser.add_argument('op', type=str,
+                             help='Snapshot operation: create or delete')
+    snap_parser.add_argument('name', type=str, action=path_to_bytes,
+                             help='Name of snapshot')
+    snap_parser.add_argument('dir', type=str, action=path_to_bytes,
+                             help='Directory for which snapshot '
+                                  'needs to be created or deleted')
+
+    @with_argparser(snap_parser)
+    def do_snap(self, args):
+        """
+        Snapshot management for the volume
+        """
+        # setting self.colors to None turns off colorizing and
+        # perror emits plain text
+        self.colors = None
```

Comment:
`perror()` emits the ANSI escapes for coloring terminal output ... and that gets in the way of parsing text on `stderr`.
Also, there's no other way to quiesce emission of color codes that I could find.

Reply:
Yeah, but just turning colouring off looks like an extreme measure. You reset the value at end of method and add a TODO. 

Anyways, I have written a PR that would allow turning colouring. You can also use regex till then (that is what I did as temporary workaround).

Reply:
@rishabh-d-dave let's clean this up in your coloring PR.